examples/09_bmg_grouped_gemm_f8/09_bmg_grouped_gemm_f8.cpp

```diff
@@ -576,19 +576,22 @@ int launcher(Options& options)
   using LayoutC = cutlass::layout::RowMajor;
   using LayoutD = cutlass::layout::RowMajor;
 
-  using GmemTiledCopyA = XE_2D_U8x32x32_LD_V;
-  using GmemTiledCopyB = XE_2D_U8x32x32_LD_V;
+  using GmemTiledCopyA = void;
+  using GmemTiledCopyB = void;
```

Comment:
Maybe leave a comment here along the lines of "'void' means to use the default copy op."